docs/other/fontobject.rst

```diff
@@ -366,3 +453,54 @@ An array of ``CTScript`` enumerated value; read-only. One or more of:
 -  ``CTScript.CT_TAILE_SCRIPT``
 -  ``CTScript.CT_TAIVIET_SCRIPT``
 -  ``CTScript.CT_DONTKNOW_SCRIPT``
+
+=======
+Methods
+=======
+
+.. _FontObject.hasSameDict:
+
+FontObject.hasSameDict()
+**********************************************
+
+``app.fonts.fontsWithDefaultDesignAxes()[0].hasSameDict(Font)``
+
+**Description**
+
+This function will true if the Font Object passed as an argument share the same variable font dictionnary.
```

Comment:
```suggestion
This function will return true if the Font Object passed as an argument shares the same variable font dictionary as the Font object the function is called on.
```

Reply:
Can this clarify what "dictionary" means in this context? And when you would want to use this.

Reply:
Font files are in one sense simply a "dictionary" which maps from characters to glyphs.

You have the character "A", you ask the font for glyph(s) which map to that character.

It is a bit more complicated than that but it is a good paradigm.

Variable fonts in particular mess with the terms because here one _physical_ font file on disk (aka a font dictionary) supports _many_ different font instances/faces.

This leads to all sorts of language problems with these new Font scripting objects since comparing two different Font object instances and asking them if they are the same..._what do you mean by same?_

With this API we are simply providing a way to compare to Font objects to see if they are mapped in some way to the same physical font file (dictionary) on disk.